src/types.ts

```diff
@@ -346,6 +391,104 @@ export interface WapeSaleInstance {
   numberPurchasableByAccount(address: string): Promise<number>;
 }
 
+export interface GenSaleInstance {
+  /** Get the price of the sale */
+  getSalePrice(): Promise<string>;
+
+  /** Get data about the current sale */
+  getSaleData(): Promise<GenSaleData>;
+
+  /** Get the block that the sale started on (will be zero unless the sale already started) */
+  getSaleStartBlock(): Promise<string>;
+
+  /** Get the current status of the sale */
+  getSaleStatus(): Promise<GenSaleStatus>;
+
+  /** Get the mint list */
+  getMintlist(): Promise<Mintlist>;
+
+  /** Sets the price of the sale */
+  setSalePrice(price: ethers.BigNumber, signer: ethers.Signer): Promise<ethers.ContractTransaction>;
+
+  /** Get a users claim from the mintlist */
+  getMintlistedUserClaim(address: string): Promise<Claim>;
+
+  /** Get how many domains for for sale (in the current phase) */
+  getTotalForSale(): Promise<ethers.BigNumber>;
```

Comment:
How big this value is? Is it effective to use BigNumber instead of a number?

Reply:
I dont think so, I just reused the types from the previous sales because I thought it might be good to preserve the old interactions with the dAPP and the sdk. I can try turning it into a number